yearn/deploy/00_deploy_cvp_strategy.ts

```diff
@@ -0,0 +1,64 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, helpers } = hre
+  const { deploy, log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const TBTCCurveVault = await deployments.get("TBTCCurveVault")
```

Comment:
Can we have a way for `local` networks to test the deployment scripts? Maybe we could fork the mainnet in hardhat network? Or use stubs? Or use fake values? Any solution that would let us test the deployment scripts on local network would be fantastic. If it's not possible with `development` network `hardhat` network will be super fine.

If we already have this possibility can we describe it in documentation?

Reply:
I tried to do this with `development` network but wasn't able to make it work. There are so many dependencies to stub and I still received failures during deployment. I will try with `hardhat`.

Reply:
It would be awesome to handle it for `hardhat` network, as now we have no way to test the scripts, except deploying them on mainnet.

Reply:
Ok, I managed to run the script on `hardhat` network: https://github.com/keep-network/tbtc-v2/pull/14/commits/299f4985dbf6794e0bd337bba96bc3eabeddaa96. I used mainnet forking and leveraged Yearn, Curve and Convex contracts existing on mainnet for tBTC v1. The two first strategies (`CurveVoterProxyStrategy` and `ConvexStrategy`) are deployed without hassle. However the script doesn't pass for `SaddleStrategy`. This is because there is no Saddle vault for tBTC v1 which can be used here. I left a TODO and we'll make it work once Saddle vault for tBTC v2 is created. I don't see any simple workaround for that until then.